index.html

```diff
@@ -10,9 +10,47 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
+    <!-- preload product images before the bundle loads -->
+    <link
+      rel="preload"
+      as="image"
+      type="image/png"
+      href="/images/stitch/1.png"
+    />
+    <link
+      rel="preload"
+      as="image"
+      type="image/png"
+      href="/images/stitch/2.png"
+    />
+    <link
+      rel="preload"
+      as="image"
+      type="image/png"
+      href="/images/stitch/3.png"
+    />
+    <link
+      rel="preload"
+      as="image"
+      type="image/png"
+      href="/images/luminara/1.png"
+    />
+    <link
+      rel="preload"
+      as="image"
+      type="image/png"
+      href="/images/luminara/2.png"
+    />
+    <link rel="preload" as="image" type="image/png" href="/images/pug/1.png" />
+    <link rel="preload" as="image" type="image/png" href="/images/pug/2.png" />
+    <link rel="preload" as="image" type="image/png" href="/images/pug/3.png" />
+
+    <!-- preload the Lato font -->
     <link
       href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
-      rel="stylesheet"
+      rel="preload"
+      as="style"
+      onload="this.onload=null;this.rel='stylesheet'"
     />
```

Comment:
This is a trick that preloads the CSS stylesheet for the Lato font first, then turns it into a stylesheet once it finishes loading. Makes the font loads a bit faster.